src/helpers.js

```diff
@@ -352,12 +354,19 @@ const getBuckets = function(data, input, aggregations) {
           filters = input.filters[k];
         }
 
+        let doc_count = v2[1].array().length;
+
+        if (hide_zero_doc_count && doc_count === 0) {
+          return;
+        }
+
         return {
           key: v2[0],
-          doc_count: v2[1].array().length,
+          doc_count: doc_count,
           selected: filters.indexOf(v2[0]) !== -1
         };
       })
+      .compact(this)
```

Comment:
Is `this` argument in `compact` necessary here ? The data is flowing form one function to next thanks to `chain`.

Reply:
Absolutely right, will update shortly.

Reply:
@cigolpl Updated